packages/kamino-sdk/src/Kamino.ts

```diff
@@ -363,60 +423,290 @@ export class Kamino {
     tokenAMint: PublicKey,
     tokenBMint: PublicKey
   ): Promise<RebalanceFieldInfo[]> => {
-    let price = await this.getPriceForPair(dex, tokenAMint, tokenBMint);
+    const price = new Decimal(await this.getPriceForPair(dex, tokenAMint, tokenBMint));
 
-    let lowerPrice: number;
-    let lowerPriceInput = fieldOverrides.find((x) => x.label == 'lowerPrice');
+    const defaultFields = getDefaultManualRebalanceFieldInfos(price);
+
+    let lowerPrice = defaultFields.find((x) => x.label == 'priceLower')!.value;
```

Comment:
I think you can rewrite this and all of the other below by making it more generic. You basically need to compare two vectors of `RebalanceFieldInfo`, and if there is a field that appears in the second one (`fieldOverrides`) override the default. Maybe you can also make them dictionaries instead of vectors so it's easier to compare.